src/server/index.jsx

```diff
@@ -104,14 +104,17 @@ server
 
       const headHelmet = Helmet.renderStatic();
 
-      const styleTags = sheet.getStyleElement();
+      const styles = sheet.getStyleElement()[0];
```

Comment:
IMO we should map over this and accumulate them in `inlineCss`, there's likely a reason its an array